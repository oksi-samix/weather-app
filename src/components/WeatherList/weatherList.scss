.icon-box{
  width: 45px;
  margin-right:15px;
}
