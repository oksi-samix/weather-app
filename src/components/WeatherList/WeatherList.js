import React from "react";
import moment from "moment";

import List from "@material-ui/core/List/List";
import ListItem from "@material-ui/core/ListItem/ListItem";
import Skycons from 'react-skycons'

import './weatherList.scss'


const WeatherList = (props) => {
  console.log(props);
  return <List>
    {props.data.map((item, index) =>
      <ListItem key={item.time + index}>
        <div className="icon-box">
          <Skycons
            color='yellow'
            icon={item.icon.replace(/-/g,'_').toUpperCase()}
          />
        </div>
        {moment.unix(item.time).add(1, 'day').format('YYYY-MM-DD')} - {item.summary}
      </ListItem>)}
  </List>
};

export default WeatherList
