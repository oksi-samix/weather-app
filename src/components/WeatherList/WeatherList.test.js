import React from 'react';
import { WeatherList } from './WeatherList';
import renderer from 'react-test-renderer';

it('renders WeatherList correctly', () => {
  const tree = renderer
    .create(<WeatherList data={[]}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
