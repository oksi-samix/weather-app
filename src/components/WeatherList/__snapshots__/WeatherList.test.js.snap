// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WeatherList correctly 1`] = `
<ul
  className="MuiList-root-1 MuiList-padding-2"
/>
`;
